python/faery/cli/__init__.py

```diff
@@ -1,8 +1,8 @@
-from . import colormaps as colormaps
-from . import convert as convert
-from . import init as init
-from . import inline as inline
-from . import render as render
-from . import run as run
+from . import colormaps
```

Comment:
The syntax `from . import colormaps as colormaps` (which is a little ugly and redundant) is unfortunately the recommended approach in __init__ files (https://github.com/microsoft/pylance-release/issues/856).